release-notes/2023-10/README.md

```diff
@@ -0,0 +1,252 @@
+                - `strimzi-kafka-operator.enabled`
+            - Run a Helm upgrade
+        5. Remove the CRDs for the Strimzi Kafka Operator from the cluster. By design, these are not removed when the operator is uninstalled, so they need to be cleaned up manually. Run `kubectl delete -f systemlinkadmin/charts/strimzi-kafka-operator/crds` to delete the CRDs.
+        6. Delete the Persistent Volume Claims (PVCs) for the Kafka-related pods. Looks for PVCs containing "dfs-kafka" in Lens.
+    - After completing these steps, if you need to update SystemLink Enterprise again, you should skip steps 2 and 3 of the updating instructions for updating the Strimzi Kafka Operator CRDs, to avoid recreating the unneeded CDs.
+- dataframeservice 0.14.39
+    - Default memory request and limit increased from 2GB per DataFrame Service pod to 4GB. Disabling Kafka (see separate instructions) will greatly reduce overall resource usage for the cluster.
+    - Since data tables created after updating won't consume additional Kafka resources, you can likely decrease the dataframeservice.kafkaconnect.spec.resources.requests.memory and/or dataframeservice.kafkacluster.kafka.resources.requests.memory values if required to fit the larger DataFrame Service pods until Kafka is disabled.
+    - Note that appendable tables created prior to the update will continue to consume Kafka resources.
+- The Test Analytics privilege category has been added, and includes the Query Measurements privilege. This privilege is not currently functional and is being added in support of features that will release in a future version.
+- testinsightsui 0.6.107
+    - Comments service was added to top level helm chart in previous release 2023-09. In this release, feature flag is removed and comments tab will be visible in result details page and user can add/view/edit/delete comments under a test result.
+
+## Helm Chart Breaking Changes
+
+- New - mongodb connection string global value override.
+    - Customers can specify the user/password directly in the global ("mongodb+srv://user:pass@host/<`database`>") (<> will be replaced during per-service Helm install/upgrade)
+    - Customers can use per-service user/password combinations ("mongodb+srv://<`username`>:<`password`>@host/<`database`>") (<> will be replaced during per-service Helm install/upgrade)
+- CRITICAL - The systemlink Helm chart defaults to connect to an external MongoDB instance
+    - If you have an existing installation of SLE you MUST etiher set `global.mongodb.install` to `true` or migrate your existing data to an external MongoDB instance and provide the connection string in `global.mongodb.connection_string`
+- dataframeservice
+    - dataframeservice.requestBodySizeLimitMegabytes was renamed to dataframeservice.requestBodySizeLimit. It now accepts units in "MiB" (Mebibytes, 1024 KiB) or in "MB" (Megabytes, 1000 KB). [https://github.com/ni/install-systemlink-enterprise/pull/165]
```

Comment:
Rather than link to a PR we should be linking to lines in the values file that show this new configuration. Refer to past release notes to see how this is structured.

Reply:
@CIakab-NI Please refer to this link https://github.com/ni/install-systemlink-enterprise/blob/4da6c60d63ef48a663e78efd9b393e41b6c40ba4/getting-started/templates/systemlink-values.yaml#L579

Reply:
@priya-ni , I've added a 'View this configuration' bullet that links to the link above, for dataframeservice.requestBodySizeLimit.

We'll need to replace the link so it uses the tag for the 2023-10 release.

Reply:
I'm marking this resolved since @priya-ni has updated the link to include the 2023-10 tag